### PR TITLE
Make Adjust Balance button functional on Subscriptions page

### PR DESCRIPTION
**Pull request by sravfeyn**
http://manage.dimagi.com/default.asp?163274

### CODE REVIEW

File: corehq/apps/accounting/forms.py [hunk 1]
```diff
@@ -1693,6 +1693,9 @@ def __init__(self, invoice, *args, **kwargs):
         self.fields['invoice_id'].initial = invoice.id
         self.helper = FormHelper()
         self.helper.form_class = "form-horizontal"
+        # circular import
+        from corehq.apps.accounting.views import InvoiceSummaryView
```
Comment (dimagimon) [inline corehq/apps/accounting/forms.py]: F401 'InvoiceSummaryView' imported but unused

File: corehq/apps/accounting/templates/accounting/partials/invoice_table.html [hunk 2]
```diff
@@ -2,7 +2,8 @@
 {% load report_tags %}
 {% load i18n %}
 
-<a href="{{invoice_report_url}}"> {% trans "Go to Invoices for this Project Space" %}</a>
+<p class="alert alert-warning">Showing All Invoices for the Subscriber's project space</p>
+<a href="{{invoice_report_url}}"> {% trans "Go to all Invoices page for this Project Space" %}</a>
```
Comment (kaapstorm) [inline corehq/apps/accounting/templates/accounting/partials/invoice_table.html]: Nit-picky, I know, but I'd make the capitalisation consistent with the alert above, i.e. "Go to All Invoices page for this project space".

**Comment by kaapstorm** [comment]
Thanks for humouring the OCD :grin:

**Comment by nickpell** [comment]
:+1:

**Pull request merged**